openslides/motions/static/js/motions/site.js

```diff
@@ -1065,16 +1183,15 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions', 'OpenSlid
     'Workflow',
     'Config',
     'motion',
-    'SingleMotionContentProvider',
     'MotionContentProvider',
     'PollContentProvider',
     'PdfMakeConverter',
     'PdfMakeDocumentProvider',
     'MotionInlineEditing',
     'gettextCatalog',
     'Projector',
-    function($scope, $http, ngDialog, MotionForm, Motion, Category, Mediafile, Tag, User, Workflow, Config,
-             motion, SingleMotionContentProvider, MotionContentProvider, PollContentProvider,
+    function($scope, $http, ngDialog, MotionComment, MotionForm, Motion, Category, Mediafile, Tag, User, Workflow, Config,
```

Comment:
actually I inserted MotionContent manually during the rebase. I thought this was new. I just gonna remove it again.